packages/amplify-graphql-function-transformer/src/__tests__/amplify-graphql-function-transformer1.test.ts

```diff
@@ -0,0 +1,29 @@
+'use strict';
```

Comment:
Can we use a different name than amplify-graphql-function-transformer1.test.ts for the test file (and it's snapshot file). It's not very descriptive.